calledudeBot/Common/Program.cs

```diff
@@ -29,14 +23,14 @@ private static void Main()
             };
             CleanCmdFile();
 
-            CredentialChecker.ProduceBots();
-            _bots = CredentialChecker.GetVerifiedBots(out discordBot, out twitchBot, out osuBot);
+            CredentialChecker.ProduceBots().Wait();
```

Comment:
`.Wait()` :stare:

Reply:
I thought I could sneak one by seeing as how `Main()` isn't really that important anyway, but sure.